src/components/twilio-voice-dialer/public/twilio-voice-dialer.js

```diff
@@ -96,7 +96,7 @@ class TwilioVoiceDialer extends HTMLElement {
   }
 
   #incomingHandler = (call) => {
```

Comment:
I think we should rename some of the methods. Just from names, it's hard to tell what `incomingHandler` and `incomingEvent` do. The other handlers are labeled `handleX`, should we do the same with `incomingHandler` and rename to `handleIncoming`? For the `incomingEvent` and `tokenWillExpireEvent`, those might be better named as `raiseIncomingEvent` and `raiseTokenWillExpireEvent`? I think methods being verbs is a good idea.

Reply:
I like this idea 👍